mlir/include/Catalyst/Transforms/Passes.td

```diff
@@ -124,4 +124,14 @@ def AddExceptionHandlingPass : Pass<"add-exception-handling"> {
     ];
 }
 
+def GEPInboundsPass : Pass<"gep-inbounds"> {
+    let summary = "Mark GEPOp inbounds.";
+
+    let dependentDialects = [
+        "mlir::LLVM::LLVMDialect",
+    ];
+
```

Comment:
We don't create any llvm ops, so the dialect is not technically a dependent dialect :)
```suggestion
```